packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -183,14 +196,12 @@ contract L2OutputOracle is Initializable, Semver {
         uint256 _l1BlockNumber
     ) external payable {
         require(
-            msg.sender == PROPOSER,
-            "L2OutputOracle: only the proposer address can propose new outputs"
+            msg.value >= OUTPUT_BOND_COST,
```

Comment:
Didn't we decide on using the simple bond scheme as seen in the [specs](https://github.com/ethereum-optimism/optimism/blob/develop/specs/bond-manager.md#simple-bond)?

Reply:
Correct, these are very old changes now